solr/core/src/test/org/apache/solr/search/TestSearchPerf.java

```diff
@@ -153,31 +155,23 @@ int doSetGen(int iter, Query q) throws Exception {
     return ret;
   }
 
-  int doListGen(int iter, Query q, List<Query> filt, boolean cacheQuery, boolean cacheFilt)
-      throws Exception {
+  int doListGen(Query q, List<Query> filt) throws Exception {
```

Comment:
Since this method is not private, need we worry about maintaining the signature? Or if changing the signature anyhow, let's make it private at the same time?
```suggestion
  private int doListGen(Query q, List<Query> filt) throws Exception {
```